### PR TITLE
updated readme and changed project name in makefile

### PR DESCRIPTION
**Pull request by ExtraGravity**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,26 @@
...
+The embedded test tool will send out signals connected to a different board, and check the signals coming back.
+The test MBED is controlled via serial and will output pass, fail, and information messages based on how the target board handled the inputs.
+
+## Compiling
+
+To compile, you will need the `arm-none-eabi-g++` compiler.
+You can get it from https://github.com/adamgreen/gcc4mbed.
+Run whichever installer script matches your OS, then run the `BuildShell` script.
+This will temporarily add the compiler to your PATH, you can then compile.
+You can use the `makeall.sh` script to compile both the tests and source and then run the tests.
```
Comment (bill-luu) [inline README.md:14]: You should specify that the makeall.sh script is in the Embedded Test Tool files, I spent forever looking through the adamgreenmbed folder haha 

Also your wording makes it seem like this should be the next step, although you it seems you need to finish the "Testing" portion below first

File: README.md [hunk 2]
```diff
@@ -1,7 +1,26 @@
...
+To only compile, simply run `make` in the root directory. 
+
+## Running
+
+After compiling the source, a `EmbeddedTestTool.bin` will be created in the root directory.
+Connect an mbed LPC1768 to your computer and copy `EmbeddedTestTool.bin` into the root directory of the MBED. 
+To interact with the program, you will need to connect to the MBED's serial port, you can do this with [PuTTy](http://www.chiark.greenend.org.uk/~sgtatham/putty/download.html).
```
Comment (bill-luu) [inline README.md:21]: How do you use putty after you download it? How do you connect to the mbed? Maybe include the sudo apt get for linux users

Also explain how you're supposed to connect to the mbed from putty

**Comment by bill-luu** [comment]
Your last line tells us to include the 'libgmock.a' file into the 'Schulich-Delta-CCS-Test-Tool/test/' directory
You have the wrong directory, it should be 'Embedded Test Tool/test/'

**Comment by ExtraGravity** [comment]
Replaced by PR #5

**Pull request closed without merging**